Gopkg.toml

```diff
@@ -528,3 +528,7 @@ required = [
     name = "k8s.io/code-generator"
     unused-packages = false
     non-go = false
+
+[[constraint]]
+  name = "github.com/davecgh/go-spew"
```

Comment:
Please add the main usage and keep the dependencies in alphabetical order 

```
 [[constraint]]
   name = "github.com/davecgh/go-spew"
   revision = "v1.1.1"
 	 
   # main-usage = "pkg/debug"
   # on-revision = ""
```

Reply:
@tgraf not addressed?

Reply:
Oh, I missed that part.